schemas/session-replay/mobile/mutation-data-schema.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "mutation-data-schema.json",
+    "title": "MutationData",
+    "type": "object",
+    "description": "Schema of a MutationData.",
+    "allOf": [
+        {
+            "properties": {
+                "source": {
+                    "type": "integer",
+                    "const": "1",
+                    "description": "The source of this type of incremental data.",
+                    "readOnly": true
+                },
+                "adds": {
+                    "type": "array",
+                    "readOnly": true,
+                    "items": {
+                        "type": "object",
+                        "required": [
+                            "previousId",
+                            "wireframe"
+                        ],
+                        "properties": {
+                            "previousId": {
+                                "type": "integer",
+                                "description": "The previous wireframe id next/after which this new wireframe should be drawn/attached to.",
+                                "readOnly": true
+                            },
```

Comment:
I'm not sure this is correct. We need the `previousId` to support `null` value in order to place new `wireframe` at the beginning of the array.

Ref. in RFC: [Adding new wireframes](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/2471460887/RFC+-+Data+Format+for+Mobile+Session+Replay)
> Besides the wireframe to add, it holds the ID of the previous wireframe in draw order (or null if the new wireframe should be rendered at the very bottom).

So, the allowed values for `previousId` should be: `Int | null`. Doesn't it mean we need to enhance the `type` definition for `previousId`? I'm not sure how to model this in [JSON schema](https://json-schema.org/understanding-json-schema/reference/object.html#id8). 

@mariusc83  @BenoitZugmeyer any thoughts?

Reply:
👍 Let's make it not `required` then. In fact, in Swift-generated code both `Int | null | none` are expressed with optional, so no preference on our side.

Reply:
Yup, good catch, we will make it not required.